apps/kg/train_pytorch.py

```diff
@@ -279,10 +275,8 @@ def dist_train_test(args, model, train_sampler, entity_pb, relation_pb, l2g, ran
             end += count
             percent += 1
 
-        if args.save_emb is not None:
-            if not os.path.exists(args.save_emb):
-                os.mkdir(args.save_emb)
```

Comment:
why remove this?

Reply:
It is not been removed. I change it to :

```
if args.save_emb is True:
            model_test.save_emb(args.save_path, args.dataset)
```

Reply:
what i mean is why removing the code of creating a directory? it's possible that the directory doesn't exist.

Reply:
I find we must use save-emb url in test. Just change it back...